### DISCUSSION TITLE
Themes

### DISCUSSION THREAD

**Discussion by hackmeister** [helix-editor/helix#12325] [Troubleshooting]
I can't get theme to display on one of my machines. When I attempt to change themes I only see two themes not a full page of themes like I should. Can someone help with this. Thanks

**Comment by TornaxO7** [comment]
Which helix version are you using?

**Comment by 4ndv** [comment]
Have you linked runtime files?

https://docs.helix-editor.com/building-from-source.html#configuring-helixs-runtime-files
